packages/garbo/src/target/fights.ts

```diff
@@ -1098,8 +1015,9 @@ export const copyTargetSources = [
 ];
 
 export function copyTargetCount(): number {
```

Comment:
Is this worth having two separate functions (or a boolean arg) for? is everywhere we call `copyTargetCount` a place where we want to include embezzlers? even if you're fighting, say, roaches?